lua/gitlab/hunks.lua

```diff
@@ -90,18 +90,28 @@ local line_was_added = function(linnr, hunk, all_diff_output)
 end
 
 ---Parse git diff hunks.
----@param file_path string Path to file.
 ---@param base_sha string Git base SHA of merge request.
 ---@return HunksAndDiff
-local parse_hunks_and_diff = function(file_path, base_sha)
+local parse_hunks_and_diff = function(base_sha)
   local hunks = {}
   local all_diff_output = {}
 
-  local Job = require("plenary.job")
+  local reviewer = require("gitlab.reviewer")
+  local cmd = {
+    "diff",
+    "--minimal",
+    "--unified=0",
+    "--no-color",
+    base_sha,
+    "--",
+    reviewer.get_current_file_oldpath(),
```

Comment:
Include the old path when calculating the diff, since the new file may have a different path